h2o-algos/src/test/java/hex/deeplearning/DeepLearningReproducibilityTest.java

```diff
@@ -19,7 +20,7 @@
 
   @Test
   public void run() {
-    NFSFileVec ff = NFSFileVec.make(find_test_file("smalldata/junit/weather.csv"));
+    NFSFileVec ff = TestUtil.makeNfsFileVec("smalldata/junit/weather.csv");
```

Comment:
This is already in master - we should not see changes here. It is impossible to review so big change.